### ISSUE TITLE
SyntaxError on running inputscope

### ISSUE THREAD

**Issue by ulcuber** [suurjaak/InputScope#17]
```
File "<stdin>", line 1
    start mouse
          ^
SyntaxError: invalid syntax
```

**Comment by suurjaak** [comment]
What was the command that resulted in this error?

**Comment by ulcuber** [comment]
@suurjaak main `inputscope`
Running `inputscope-*` works

**Comment by suurjaak** [comment]
Could you post a screenshot?

**Comment by ulcuber** [comment]
@suurjaak 
![image](https://user-images.githubusercontent.com/32728904/178033961-53a592b3-0c30-4233-8ec3-43f00594698c.png)

**Comment by ulcuber** [comment]
@suurjaak and then cannot exit via ctrl+c
![image](https://user-images.githubusercontent.com/32728904/178034274-9814bc92-9e90-4f9b-9661-7be15c48a366.png)
![image](https://user-images.githubusercontent.com/32728904/178034384-d0bae368-8a54-4d6a-af36-42f232ac61f4.png)
```
Traceback (most recent call last):
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 362, in OnLogResolution
    def OnLogResolution(self, event=None):
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 66, in <lambda>
    signal.signal(signal.SIGINT,   lambda *a, **kw: self.stop(True))
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 112, in stop
    if self.listener: self.listenerqueue.put("exit"), self.listener.terminate()
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/util.py", line 115, in put
    self.output.flush()
BrokenPipeError: [Errno 32] Broken pipe
Traceback (most recent call last):
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 362, in OnLogResolution
    def OnLogResolution(self, event=None):
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 66, in <lambda>
    signal.signal(signal.SIGINT,   lambda *a, **kw: self.stop(True))
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 112, in stop
    if self.listener: self.listenerqueue.put("exit"), self.listener.terminate()
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/util.py", line 115, in put
    self.output.flush()
BrokenPipeError: [Errno 32] Broken pipe
Traceback (most recent call last):
  File "/home/victor/.local/lib/python3.9/site-packages/wx/py/shell.py", line 383, in OnIdle
    def OnIdle(self, event):
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 66, in <lambda>
    signal.signal(signal.SIGINT,   lambda *a, **kw: self.stop(True))
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/main.py", line 112, in stop
    if self.listener: self.listenerqueue.put("exit"), self.listener.terminate()
  File "/home/victor/.local/lib/python3.9/site-packages/inputscope/util.py", line 115, in put
    self.output.flush()
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by suurjaak** [comment]
Can you tell me more about your setup:

1. What operating system are you using? 
2. How was InputScope installed?

**Comment by ulcuber** [comment]
@suurjaak 
1.
- OS: Gentoo Base System release 2.8 x86_64 
- Kernel: 5.10.76-gentoo-r1
2. `pip install --user inputscope`
And from source `pip install --user ./` same output

**Comment by suurjaak** [comment]
Where do those stack traces originate from? It sort of looks like they're from InputScope's Python console?

**Comment by ulcuber** [comment]
https://github.com/suurjaak/InputScope/issues/17#issuecomment-1179178145
That seems like error communicating with subprocess

https://github.com/suurjaak/InputScope/issues/17#issuecomment-1179179600
That one probably doing too much in `signal.signal(signal.SIGINT,   lambda *a, **kw: self.stop(True))`. Too heavy handler for signal. Usually they only setting flag like `self.running = False`

**Comment by suurjaak** [comment]
Do I understand the situation correctly:

1. On running `inputscope`, the SyntaxError gets printed, but the program itself continues to run?
2. On pressing Ctrl-C, you get the stack trace?

**Comment by ulcuber** [comment]
@suurjaak
1. Yes, continues but not tracks events.
2. Yes.
Program itself shows icon in tray and able to open web server. Main problem affects only listener.

**Comment by suurjaak** [comment]
Fixed in the freshly released v1.6.
